strings/tips/xeno.txt

```diff
@@ -1,66 +1,65 @@
+As a boiler, the King or the Queen, you have the most powerful corrosive acid for melting things.
+As the Shrike, take advantage of your psychic abilities. Use Psychic Fling and Unrelenting Force to displace a marine or even push certain and hazardous objects towards your enemies and vice versa.
 As the Queen, your screech does not only affect the mobs on view, but affects through walls or gases. However, the effect of the screech will be reduced.
-As a drone, hivelord, hivemind, shrike, and queen, a large maze is almost always a good investment.
-As a drone, hivelord, hivemind, shrike, and queen, be clever with your walling. Try to allow for as much fluid movement as possible.
+As a drone, hivelord, Hivemind, Shrike, and the Queen, a large maze is almost always a good investment.
+As a drone, hivelord, Hivemind, Shrike, and the Queen, be clever with your walling. Try to allow for as much fluid movement as possible.
 Sticky resin can act as the path for your maze to be built around.
 The Hivemind is very skilled at flanking marines. Wait for them to push, and wall them off from behind!
 Praying to the Queen-Mother will usually net a response.
 Your queen is your lifeline. Try to listen to her.
 As a hunter, the movement of your Mirage is controlled by your intent.
 Defenders have a very small window to sweep grenades back at marines with their tail.
 Aiming for the head is a good long-term investment, causing brain damage on marines.
-Aiming for the groin can easily damage someones liver.
+Aiming for the chest can easily damage someone's liver, lungs, or even heart.
 Arms and legs are particularly vulnerable to damage.
 As a xeno, to be robust is to know when to disengage.
 As a xeno, in a duel, observe your enemy. They are likely predictable.
-The QM has put a lot of faith into your hive. It would be a shame to let her down.
+The Queen-Mother has put a lot of faith into your hive. It would be a shame to let her down.
 As a xeno, attack between the delay of a shotgun being fired.
-As a xeno, listen to the sound of your enemies magazine! If it sounds hollow, it will soon be empty.
+As a xeno, listen to the sound of your enemy's magazine! If it sounds hollow, it will soon be empty.
 As a xeno, strike hard and fast.
-As a xeno, you are not disposable. However, sacrificing your life to save the queen is widely respected.
+As a xeno, you are not disposable. However, sacrificing your life to save the Queen is widely respected.
 A fortified defender is one of the best ways to stall a push.
 Skilled defilers can easily kill any marine in a span of seconds.
 As a xeno, always have an escape plan.
 Tileswapping marines via help intent can have interesting consequences.
 As a boiler, you'll begin to glow with the more globules you have stored.
 If you see a red dot, RUN!
 Allowing marines to push caves only to block off their exit is hillariously effective.
-As a xeno, a Tier 3 unit takes up both a Tier 3 and Tier 2 slot.
```

Comment:
This is true, why'd you delete?

Reply:
i checked the code but didn't see anything for it, will add back